### ISSUE TITLE
Memory Free Percentage Sensor is actually Showing Memory Used

### ISSUE THREAD

**Issue by birdwing** [dougiteixeira/proxmoxve#25]
![image](https://user-images.githubusercontent.com/6484397/211060682-22063a87-dc53-432c-a317-aac274d58084.png)

Here Memory Free + Memory Used = 15.1
Memory Free is 6.12 which is only 40.53% of the total (15.1) however Memory free percentage says 59.5%

Memory Used is 8.98 which is 59.5% of the total (15.1) which is what Memory free percentage says.
So the *Memory Free Percentage* sensor is really a *Memory Used Percentage*

**Comment by birdwing** [comment]
The issue is found in the sensor.py file.

Lines 87 - 96
```python
    ProxmoxSensorEntityDescription(
        key="memory_free_perc",
        name="Memory free percentage",
        icon="mdi:memory",
        native_unit_of_measurement=PERCENTAGE,
        conversion_fn=lambda x: round(x * 100, 1),
        value_fn=lambda x: 1
        - x[ProxmoxKeyAPIParse.MEMORY_FREE] / x[ProxmoxKeyAPIParse.MEMORY_TOTAL],
        state_class=SensorStateClass.MEASUREMENT,
    ),
```

The value_fn  has a 1 - FREE / TOTAL, the  " 1 - " inverts the percentage.
removing that fixes the issue:

```python
    ProxmoxSensorEntityDescription(
        key="memory_free_perc",
        name="Memory free percentage",
        icon="mdi:memory",
        native_unit_of_measurement=PERCENTAGE,
        conversion_fn=lambda x: round(x * 100, 1),
        value_fn=lambda x: x[ProxmoxKeyAPIParse.MEMORY_FREE] / x[ProxmoxKeyAPIParse.MEMORY_TOTAL],
        state_class=SensorStateClass.MEASUREMENT,
    ),
```

**Comment by dougiteixeira** [comment]
Corrigido por #26.

**Comment by kzaoaai** [comment]
still an issue.
![image](https://user-images.githubusercontent.com/39029008/222609388-d369f15b-21ea-4545-8897-b39fde6cc48e.png)

**Comment by dougiteixeira** [comment]
> still an issue. ![image](https://user-images.githubusercontent.com/39029008/222609388-d369f15b-21ea-4545-8897-b39fde6cc48e.png)

Only node sensors had been fixed, for VMs it was fixed in #44. Thanks!

**Comment by kzaoaai** [comment]
Great, thank you!
